pkg/controller/labels.go

```diff
@@ -41,7 +42,7 @@ func getLabelsToEnsureFromMessage(message string) []string {
 
 func labelSliceContains(slice []*github.Label, s string) bool {
 	for _, element := range slice {
-		if element.GetName() == s {
+		if strings.Compare(element.GetName(), s) == 0 {
```

Comment:
Why did you change this? From the `go doc`:

```
Compare is included only for symmetry with package bytes. It is usually
    clearer and always faster to use the built-in string comparison operators
    ==, <, >, and so on
```

Reply:
what? ;)  - fixed